src/actions/index.jsx

```diff
@@ -1,22 +1,28 @@
-
 export const FILTER = 'FILTER';
-export const DELETE = 'DELETE'
+export const DELETE = 'DELETE';
 
-export const filters = (value) => {
-  return {type: FILTER, payload: value}
-}
+export const filters = (value) => ({ type: FILTER, payload: value });
+export const deleteItem = (data) => ({ type: DELETE, payload: data });
 
 export function doFilter(data, pattern) {
```

Comment:
Oh I see, I think we should use Array.filter() function instead of forEach(), how do you think?